agent/stats/service_connect_linux_test.go

```diff
@@ -43,26 +45,94 @@ func TestRetrieveServiceConnectMetrics(t *testing.T) {
 		LocalIPAddressUnsafe: "127.0.0.1",
 	}
 
-	// Set up a mock http sever on the statsUrlpath
-	statsMockurl := "127.0.0.1:9901"
-	r := mux.NewRouter()
-	r.HandleFunc("/stats/prometheus", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintln(w, "Service Connect Stats")
-	}))
+	var tests = []struct {
+		stats         string
+		expectedStats []*ecstcs.GeneralMetricsWrapper
+	}{
+		{
+			stats: "# TYPE MetricFamily1 counter\nMetricFamily1{dimensionA=\"value1\", dimensionB=\"value2\"} 1\n" +
```

Comment:
nit: can we use multi-line strings to better visualize the inputs?

Reply:
MetricCounts: []*int64{aws.Int64(1), aws.Int64(1), aws.Int64(1)} - The histogramMetricCounts cumulative counts got converted into CW disjoint type.